fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/FunctionBinder.java

```diff
@@ -96,6 +96,12 @@ public Expression visitUnboundFunction(UnboundFunction unboundFunction, Cascades
                 .build()
                 : (List) unboundFunction.getArguments();
 
+        ArithmeticFunctionBinder functionBinder = new ArithmeticFunctionBinder();
```

Comment:
add comment to explain why these founction cannot be treat as normal scalar function